Eliminar usuarios con 0 estrellas de ranking-#177

jcbuildr opened the pull request:
Eliminados los usuarios de la lista y añadido título a la vista de ranking para móvil.

jcbuildr committed:
Eliminar usuarios con 0 estrellas de ranking-#177

Eliminados los usuarios de la lista y añadido título a la vista de ranking para móvil.

jorandmol reviewed (changes requested):
Deben pasar todos los test

jcbuildr committed:
Corregir test de ranking-#177

Corregido el setUp del test de ranking, no se estaban guardando los puntos de los usuarios.

juacormun commented:
Ya he corregido el test de ranking que fallaba por no tener guardados los puntos de los usuarios en el setUp.

fragonlug reviewed (approved):
Se ha validado la solución presentada ante la incidencia.
